lib/osf-components/addon/components/draft-registration-card/styles.scss

```diff
@@ -38,14 +38,16 @@
     }
 }
 
-.DraftRegistrationCard__review {
+// stylelint-disable selector-no-qualifying-type
+a.DraftRegistrationCard__review {
     margin-right: 10px;
 
     &:hover {
```

Comment:
Did the `a:hover` you had before not work?

Reply:
doing `a:hover` here would select
```
<element class="DraftRegistrationCard__review">
  …
    <a :hover>
```
I tried fiddling around trying stuff like `&a:hover` or `a &:hover`, but to no avail